cl/simple_pages/views.py

```diff
@@ -378,24 +378,37 @@ async def contact(
                 logger.info("Detected spam message. Not sending email.")
                 return HttpResponseRedirect(reverse("contact_thanks"))
 
+            issue_type_label = form.get_issue_type_display()
+
             default_from = settings.DEFAULT_FROM_EMAIL
             message = EmailMessage(
                 subject="[CourtListener] Contact: "
                 "{phone_number}".format(**cd),
                 body="Subject: {phone_number}\n"
-                "From: {name} ({email})\n\n\n"
+                "From: {name}\n"
+                "User Email: <{email}>\n"
+                "Issue Type: {issue_type_label}\n\n"
                 "{message}\n\n"
                 "Browser: {browser}".format(
                     browser=request.META.get("HTTP_USER_AGENT", "Unknown"),
+                    issue_type_label=issue_type_label,
                     **cd,
                 ),
-                to=["info@free.law"],
+                to=["support@freelawproject.atlassian.net"],
                 reply_to=[cd.get("email", default_from) or default_from],
             )
             await sync_to_async(message.send)()
             return HttpResponseRedirect(reverse("contact_thanks"))
     else:
         # the form is loading for the first time
+        issue_type = request.GET.get("issue_type")
+        if issue_type:
+            issue_type_lower = issue_type.lower()
+            valid_issue_types = [
+                choice[0] for choice in ContactForm.ISSUE_TYPE_CHOICES
+            ]
+            if issue_type_lower in valid_issue_types:
+                initial["issue_type"] = issue_type_lower
```

Comment:
if we apply the previous suggestion, we can refactor this code like this:

```suggestion
        if issue_type and issue_type.lower() in ContactForm.VALID_ISSUE_TYPES:
            initial["issue_type"] = issue_type.lower()
```